### DISCUSSION TITLE
怎么更新软件并保留原来的数据

### DISCUSSION THREAD

**Discussion by ghost** [Planshit/Tai#141] [Q&A]
把新的安装包文件夹里文件全部复制粘贴到旧的文件夹里面吗？

**Comment by noberumotto** [comment]
在软件的设置>关于中点击检查更新可以一键更新

**Comment by ghost** [comment]
那样会更新的很慢吧😥。我其实是想问把新的安装压缩包下载下来之后怎么整，是直接把里面的文件复制粘贴到旧的文件夹，然后选择全部替换吗？
________________________________
发件人: Berumotto ***@***.***>
发送时间: 2023年3月7日 21:39
收件人: Planshit/Tai ***@***.***>
抄送: Benjamin Watson ***@***.***>; Author ***@***.***>
主题: Re: [Planshit/Tai] 怎么更新软件并保留原来的数据 (Discussion #141)


在软件的设置>关于中点击检查更新可以一键更新

**Reply by noberumotto** [reply]
对的，全部替换掉，数据不会受到影响的

**Reply by 202315903** [reply]
我受到了影响

**Comment by ghost** [comment]
谢谢你，我明白了。😉
________________________________
发件人: Berumotto ***@***.***>
发送时间: 2023年3月9日 12:56
收件人: Planshit/Tai ***@***.***>
抄送: Benjamin Watson ***@***.***>; Author ***@***.***>
主题: Re: [Planshit/Tai] 怎么更新软件并保留原来的数据 (Discussion #141)


对的，全部替换掉，数据不会受到影响的
